docs/pypi-replication.md

```diff
@@ -70,6 +70,36 @@ To evaluate:
 $ wget -O jtreceval-0.0.5-jar-with-dependencies.jar https://search.maven.org/remotecontent?filepath=uk/ac/gla/dcs/terrierteam/jtreceval/0.0.5/jtreceval-0.0.5-jar-with-dependencies.jar
 $ wget https://raw.githubusercontent.com/castorini/anserini/master/src/main/resources/topics-and-qrels/qrels.msmarco-doc.dev.txt
 $ java -jar jtreceval-0.0.5-jar-with-dependencies.jar -m map -m recall.1000 qrels.msmarco-doc.dev.txt run.msmarco-doc.txt
+map                   	all	0.2310
+recall_1000           	all	0.8856
+```
+
+## Note
+- Replicate above experiments with `pyserini==0.10.0.1` are expected to get exactly same result.
+- To replicate with `pyserini==0.10.0.0`, please change indexes with `msmarco` prefix to `ms-marco`. 
+  E.g., `msmarco-doc` should be change to  `ms-marco-doc`. Exactly same results are expected to obtain.
+  
+- WARNING: In `pyserini==0.10.0.0`, hits are hard coded to 1000. see [here](https://github.com/castorini/pyserini/blob/pyserini-0.10.0.0/pyserini/search/__main__.py#L110).
+  Even though for MS MARCO doc we only evaluate top 100.
+  In `pyserini==0.10.0.1`, the hits are has been parameterized. see [here](https://github.com/castorini/pyserini/blob/pyserini-0.10.0.1/pyserini/search/__main__.py#L112).
+  
+CAVEAT: current HEAD of master is getting different results.
+```
+MS MARCO passage ranking task, BM25 baseline:
+map                   	all	0.1958
```

Comment:
Wait, aren't these matching the numbers above at (a) ^^^^?

Reply:
no, we should compare to the scores here, https://github.com/castorini/pyserini/blob/master/docs/pypi-replication.md

Reply:
Oh shoot, you're right. The difference comes from different default parameters...

ARG, this is annoying, but this is fine as is - I'll circle back and redo once we publish `0.10.0.2`.

Thanks for the replication.